### PR TITLE
optimize performance and improve concurrency

### PR DESCRIPTION
**Pull request by cyjseagull**
1. Delete frequently output logs
2. Reduce time-consuming accout_exists call

### CODE REVIEW

File: libprecompiled/Precompiled.h [hunk 1]
```diff
@@ -80,8 +80,13 @@ class Precompiled : public std::enable_shared_from_this<Precompiled>
         return m_precompiledExecResultFactory;
     }
 
+    // for UT
+    static void clearName2SelectCache();
+
 protected:
     std::map<std::string, uint32_t> name2Selector;
+    // global function selector cache
+    static tbb::concurrent_unordered_map<std::string, uint32_t> s_name2SelectCache;
```
Comment (bxq2011hust) [inline libprecompiled/Precompiled.h:89]: I think this can be deleted, just a static s_name2SelectCache in cpp is fine

Comment (cyjseagull) [reply libprecompiled/Precompiled.h:89]: ok~I will fix it.

File: libprecompiled/Precompiled.h [hunk 2]
```diff
@@ -80,8 +80,13 @@ class Precompiled : public std::enable_shared_from_this<Precompiled>
         return m_precompiledExecResultFactory;
     }
 
+    // for UT
+    static void clearName2SelectCache();
```
Comment (bxq2011hust) [inline libprecompiled/Precompiled.h:84]: if s_name2SelectCache is not static, this function can not static

Comment (cyjseagull) [reply libprecompiled/Precompiled.h:84]: Done.

**Pull request merged**